publication-event-handlers/src/test/java/no/unit/nva/publication/events/handlers/expandresources/ExpandDataEntriesHandlerTest.java

```diff
@@ -99,7 +100,7 @@ void shouldNotProduceAnExpandedDataEntryWhenInputHasNoNewImage() throws IOExcept
         var request = emulateEventEmittedByDataEntryUpdateHandler(oldImage, DELETED_RESOURCE);
         expandResourceHandler.handleRequest(request, output, CONTEXT);
         var response = parseHandlerResponse();
-        assertThat(response, is(equalTo(emptyEvent())));
+        assertThat(response, is(equalTo(emptyEvent(response.getTimestamp()))));
```

Comment:
The EventReference contains now a timestamp and so we have to inject the timestamp of the actual event to the expected event.